src/mds/Server.cc

```diff
@@ -5722,6 +5722,32 @@ int Server::parse_layout_vxattr(string name, string value, const OSDMap& osdmap,
   return 0;
 }
 
+int64_t Server::return_bytes(string& value_str)
+{
+  if (value_str.length() < 1) {
+    return -CEPHFS_EINVAL;
+  }
+  unordered_map<char, int64_t> size_value = {{'k', 1L << 10}, {'m', 1L << 20}, 
```

Comment:
The small case is used here. Is it the standard followed across the ceph project? Could you please double check?

Reply:
im not sure about the standard but I kept it like this because im converting the last character to smallcase so `100k/100K` can be mapped easily.

Reply:
I was more concerned with Kib vs KB thing. Since it's mentioned the doc, I think we are good here.